src/Parser/GuardianDruid/Modules/Features/AlwaysBeCasting.js

```diff
@@ -25,6 +28,33 @@ class AlwaysBeCasting extends CoreAlwaysBeCasting {
     SPELLS.MASS_ENTANGLEMENT_TALENT.id,
     SPELLS.WILD_CHARGE_TALENT.id,
   ];
+
+  suggestions(when) {
+    const deadTimePercentage = this.owner.modules.alwaysBeCasting.totalTimeWasted / this.owner.fightDuration;
+    
+    when(deadTimePercentage).isGreaterThan(0.2)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest(<span> Your dead GCD time can be improved. Try to Always Be Casting (ABC)..</span>)
```

Comment:
Some additional suggestions could help here, e.g. "try to reduce the intervals between casting spells while the GCD is down, and while moving use <these spells>"